doc/source/parameters/dem/floating_mesh.rst

```diff
@@ -59,7 +51,7 @@ This subsection explains the solid objects (floating meshes) information. First
 
 * In the subsection ``angular velocity``, we define the angular velocity of the floating mesh.
 
-* In the subsection ``center of rotation``, we define the center of rotation of the solid object (in the case of rotational motion).
+* The ``center of rotation`` defines the center of rotation of the solid object (in the case of rotational motion).
```

Comment:
I'm not sure about it, but maybe the center of rotation should also be a function.
What happens when your mesh moves in x but the rotation axis is in the xy plan? (let say reproducing a rolling cylinder on a surface)

Reply:
Thanks I get it now, I just did not know that the point was translated

Reply:
Yeah otherwise it's a pain in the ass to code
It could be good to add a note that says that the center of rotation is translated with the translational velocity